src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
-
+import java.util.Random;
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[randomIndex];
+        return color.toString();
     }
```

Comment:
Avoid creating a new instance of `Random` every time the method is called. Instead, create a single instance as a class-level variable. Also, use `name()` instead of `toString()` to get the string representation of the enum constant to ensure you get the actual name of the enum constant and not the overridden `toString()` representation.